Add FarmCoin (FARM) metadata for verification

VishnuS003 opened the pull request:
FarmCoin (FARM) is the in-game currency of the BUTON TAP ecosystem.
- Added metadata in "jettons/farmcoin.yaml"
- Included logo and description

VishnuS003 committed:
Create farmcoin.yaml

VishnuS003 committed:
Update farmcoin.yaml

Irso007 commented:
Make sure you have entered the correct token address. We do not see a token with this address EQCUnf4TP73HSk5_JtkH3B8nxYc4qcwfk1jA_E_0mjnPHZ

VishnuS003 committed:
Update farmcoin.yaml

VishnuS003 committed:
Update farmcoin.yaml

VishnuS003 commented:
Fixed minter address (typo), updated YAML accordingly.
This is the Jetton master address of FarmCoin: EQCLUnf4TP73HSk5_JtkH3B8nxYc4qcwfk1jA_E_0mjnPHZ. Please re-check

Irso007 commented:
Token with such address not found https://tonviewer.com/EQCLUnf4TP73HSk5_JtkH3B8nxYc4qcwfk1jA_E_0mjnPHZ

VishnuS003 committed:
Update farmcoin.yaml

VishnuS003 commented:
Hello,
We added FarmCoin (FARM) token.
Jetton master: EQCLUnf4TP73HSk5_JtkH3B8nxYc4qcwfk1jA_E_0mnjnPHZ
Example jetton wallet: https://tonviewer.com/UQC-c2mEXlcthz-BMBam6-EM6Aiok5E0Lg22zxhQyZstfhFj/jetton/EQCLUnf4TP73HSk5_JtkH3B8nxYc4qcwfk1jA_E_0mnjnPHZ?utm_source=tonkeeper
The YAML file points exactly to the master address.  
Thank you 🙏

Irso007 commented:
Uninformative or underdeveloped website/social media. Try to verify again when your website/social media is more informative and when the token has more holders.
